tests/api/test_cluster.cpp

```diff
@@ -84,15 +83,18 @@ TEST(ApiClusterTest, DifferentConstructors) {
     umd_cluster = std::make_unique<Cluster>(sdesc_path, target_devices);
     umd_cluster = nullptr;
 
-    // 4. Constructor for creating a cluster with mock chip.
-    umd_cluster = Cluster::create_mock_cluster();
-    umd_cluster = nullptr;
+    // TODO: This doesn't work at the moment.
```

Comment:
open an issue as a reminder to reenable this

Reply:
It already has to be done as part of https://github.com/tenstorrent/tt-umd/issues/372